doc-content/enterprise-only/admin-and-config/manage-service-tasks-proc.adoc

```diff
@@ -2,7 +2,7 @@
 
 = Managing custom tasks in {CENTRAL}
 
-Custom tasks (work items) are tasks that you can customize and reuse across multiple business processes or across all projects in {CENTRAL}. {PRODUCT} provides a set of custom tasks within the custom task repository in {CENTRAL}. You can enable or disable the default custom tasks and upload custom tasks into {CENTRAL} to implement the tasks in the relevant processes.
+Custom tasks (work items) are tasks that can run custom logic. You can customize and reuse custom tasks across multiple business processes or across all projects in {CENTRAL}. You can also add custom elements in the designer palette, including name, icon, sub-category, input and output parameters, and documentation. {PRODUCT} provides a set of custom tasks within the custom task repository in {CENTRAL}. You can enable or disable the default custom tasks and upload custom tasks into {CENTRAL} to implement the tasks in the relevant processes.
```

Comment:
>> Regarding _You can also add custom elements in the designer palette, including name, icon, sub-category, input and output parameters, and documentation._

If the name, icon, sub-category and so on present on the UI maybe you can bold them? wdyt?

Reply:
Good catch. These are not referencing specific UI elements so they are ok not being bolded imo. Thank you, Kalyani!